themes/socialbase/templates/layout/region--footer.html.twig

```diff
@@ -1,7 +1,24 @@
 {% if content %}
   <footer class="site-footer" role="contentinfo">
     <div class="container row">
-      {{ content }}
+
+      {% if style == 'sky' %}
```

Comment:
A few things to consider here.

How can this be best tested? It seems that our default Open Social installation has no content in this region so it's not displayed at all.

We've already said before that it's sometimes weird that the default flavour does not have a footer. Perhaps we should add this to both flavours to keep this consistent.

We've hardcoded the Open Social logo here to link home. We may want to use the theme's logo instead, or make this configurable entirely separately so different logo's are possible for header and footer. If this is supposed to be a "Powered by Open Social" block then it should not link to home but to Open Social's marketing website. We should probably also just turn it into a block in that case.

Reply:
I revert all changes, but left all styles for sky theme. Need some work from BE side with logo.